lit_gpt/config.py

```diff
@@ -1243,7 +1243,7 @@ def norm_class(self) -> Type:
     )
 ]
 for c in tiny_llama:
-    for kind, hf_postfix in (("", "-intermediate-step-955k-token-2T"), ("", "-intermediate-step-1431k-3T"), ("chat", "-Chat-v0.6"), ("chat", "-Chat-v1.0")):
+    for kind, hf_postfix in (("", "-intermediate-step-1431k-3T"), ("chat", "-chat-v1.0")):
```

Comment:
I realized that I've explained it a bit ambiguously 😊 Sorry for that.
Here is what I've meant:

```suggestion
    for kind, hf_postfix in (("", "-intermediate-step-1431k-3T"), ("-chat", "-Chat-v1.0")):
```